comp/volunteers/brief-volunteers.yaml

```
@@ -1,18 +1,35 @@
-summary: Brief Volunteers on Competition Event
-
-priority: major
-
-component: Competition
-
-milestone: $SRYYYY Competition
-
-area-owner: volunteer-coordination
-
-description: >-
-  The volunteers need to know:
-    - Key Roles & Point of Contacts
-    - Health and Safety
-      - COVID-19 Requirements
-    - Safeguarding
-    - Venue layout (if applicable)
-    - Schedule of Event
+summary: Brief volunteers at the $SRYYYY competition
+
+priority: should
+
+component: Competition
+
+milestone: $SRYYYY Competition
+
+area-owner: volunteer-coordination
+
+description: >-
+
+  We'll need to brief volunteers at the event.
+
+  The brefing should cover at a minimum:
+    - Key Roles & Point of Contacts
+    - Health and Safety
+      - COVID-19 Requirements
+    - Safeguarding
+    - Venue layout (if applicable)

```

Including venue layout as standard probably makes sense

```suggestion
    - Venue layout

```